Library/Formula/postgresql.rb

```diff
@@ -45,6 +45,10 @@ class Postgresql < Formula
   def install
     ENV.libxml2 if MacOS.version >= :snow_leopard
 
+    ENV.prepend "LDFLAGS", "-L#{Formula['openssl'].opt_lib}"
+    ENV.prepend "CFLAGS", "-L#{Formula['openssl'].opt_lib}"
```

Comment:
Did you mean `CFLAGS` here? CFLAGS normally point at headers rather than the libraries.